inc/class-wpseo-image-utils.php

```diff
@@ -128,8 +128,9 @@ public static function get_absolute_path( $path ) {
 			$uploads = wp_get_upload_dir();
 		}
 
-		if ( empty( $uploads['error'] ) ) {
-			return $uploads['basedir'] . "/$path";
+		// Only add basedir if it does not already exist in the path.
```

Comment:
This description might be wrong, because when the result of `strpos` is 0 the basedir will exist in the path.